frontend/__tests__/components/Common/DateRangeViewer/DateRangeViewer.test.tsx

```diff
@@ -42,4 +52,19 @@ describe('DateRangeViewer', () => {
     expect(getByText(/2024\/03\/19/)).toBeInTheDocument();
     expect(getByText(/2024\/03\/21/)).toBeInTheDocument();
   });
+
+  it('should show priority high icon when click expand button and step number is 1', async () => {
+    store.dispatch(nextStep());
+    store.dispatch(updateFailedTimeRange(['1706716800000']));
+    const { getByLabelText } = setup(mockDateRanges);
+    await userEvent.click(getByLabelText('expandMore'));
+    expect(screen.getByTestId('PriorityHighIcon')).toBeInTheDocument();
+  });
+
+  it('should not show priority high icon when click expand button and step number is 0', async () => {
+    store.dispatch(backStep());
```

Comment:
Among test cases, they should be as stateless as possible, hence, better to reset the store before/after each, and new backStep here.
Imagine that someone accidently add a case and have some dependency with step, it probably fails.